code/HUXt_example.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Apr 25 11:29:37 2020
+
+@author: mathewjowens
+"""
+
+import numpy as np
+import HUXt as H
+import astropy.units as u
+import matplotlib.pyplot as plt
+import time
+
+
+# <codecell> HUXt1D, no CME
+###############################################################################
+#Run HUXt1D with user specified boundary conditions and no CMEs
+###############################################################################
+
+#Form longitudinal boundary conditions - background wind of 400 km/s with two fast streams.
+v_boundary = np.ones(128) * 400 * (u.km/u.s)
+v_boundary[30:50] = 600 * (u.km/u.s)
+v_boundary[95:125] = 700 * (u.km/u.s)
+
+# This boundary condition looks like
+fig, ax = plt.subplots(figsize=(10,5))
+ax.plot(v_boundary,'k-')
+ax.set_xlabel('Longitude bin')
+ax.set_ylabel('Input Wind Speed (km/s)')
+ax.set_xlim(0,128)
+
+# Setup HUXt to do a 5 day simulation, with model output every 4 timesteps (roughly half and hour time step), looking at 0 longitude
+model = H.HUXt1D(v_boundary=v_boundary, lon=0.0*u.deg, simtime=5*u.day, dt_scale=4, rmin=30*u.solRad)
+
+# Solve these conditions, with no ConeCMEs added.
+cme_list = []
+
+#model.generate_v_boundary_ts(cme_list)
+model.solve_opt(cme_list)
```

Comment:
I wonder if we should either just replace solve() with solve_opt(), or in the examples show them both side by side. Maybe it might make sense to rename solve_opt to solve_fast or something more obvious?